{
  "modified": "2024-10-24T01:01:59Z",
  "published": "2024-06-25T13:38:19Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5499",
  "aliases": [
    "SNYK-PYTHON-PIPCOLOURADDONSV1-5932285"
  ],
  "summary": "Malicious code in pipcolouraddonsv1 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pipcolouraddonsv1",
        "purl": "pkg:pypi/pipcolouraddonsv1"
      },
      "versions": [
        "1.1.0",
        "1.0.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PIPCOLOURADDONSV1-5932285"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "47ea2e0a35db34b3b6ea4660d34ff2d94e50ec53f946a85cce8b3496eaba42a8",
        "import_time": "2024-06-28T02:49:41.616641964Z",
        "id": "RLMA-2024-04281",
        "modified_time": "2024-06-25T13:38:19Z",
        "versions": [
          "1.1.0"
        ]
      },
      {
        "source": "reversing-labs",
        "sha256": "fcd253b6c8f76e1f6df57b16e9e0dd9c9d146894af30c56b39e4505756d42eb3",
        "import_time": "2024-10-24T00:59:36.800994687Z",
        "id": "RLUA-2024-08709",
        "modified_time": "2024-10-16T14:45:40Z",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  }
}
